lib/core/src/apiHandler.cpp

```diff
@@ -109,33 +102,45 @@ namespace irods {
             return PASS( ret );
         }
 
+        namespace fs = boost::filesystem;
+
+        static std::vector<fs::path> previously_loaded_client_plugin_names;
+        static std::vector<fs::path> previously_loaded_server_plugin_names;
+        auto& loaded_plugin_names = _cli_flg ? previously_loaded_client_plugin_names
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'loaded_plugin_names' can be reduced.](https://app.codacy.com/manual/trel/irods/pullRequest?prid=5204898)